browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -81,29 +106,18 @@ final class WelcomeViewController: UIViewController {
             }
         }
     }
-    
-    func signIn() {
-        auth?.signIn(from: view.window) { result in
-            switch result {
-            case .success(let token):
-                do {
-                    try Credential.store(token)
-                    self.performSegue(withIdentifier: "show-details", sender: self)
-                } catch {
-                    self.show(title: "Error", error: error.localizedDescription, after: 3.0)
-                    return
-                }
-            case .failure(let error):
-                self.show(title: "Error", error: error.localizedDescription)
-                return
-            }
-        }
-    }
 
     @IBAction private func signInTapped() {
         auth?.ephemeralSession = ephemeralSwitch.isOn
         let isBiometricsEnabled = biometricStorageSwitch.isOn
-        isBiometricsEnabled ? SignInWithBiometrics() : signIn()
+        
+        // This demonstrates how to securely store the sign in credentials behind a biometric factor
+        // or using the default implementation provided. Only choose one approach or the other and not both.
```

Comment:
I'd suggest moving these to the overall `signInTapped` function.  You can describe how it uses the biometrics switch to choose one approach or another.  You can also use the double-back tick notation to cross-reference to the appropriate function.